tempesta_fw/Makefile

```diff
@@ -16,7 +16,7 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59
 # Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 
-EXTRA_CFLAGS += -Werror -I$(src)/../sync_socket -I$(src)/../tempesta_db
+EXTRA_CFLAGS +=  -I$(src)/../sync_socket -I$(src)/../tempesta_db
```

Comment:
Perhaps it would be better to keep `-Werror` there?
At some point we agreed that all compiler warnings shall be fixed, so this option facilitates that.